javaparser-core-testing/src/test/java/com/github/javaparser/ast/comments/CommentTest.java

```diff
@@ -69,4 +77,125 @@ public void unicodeEscapesArePreservedInComments() {
         Comment comment = cu.getAllContainedComments().get(0);
         assertEquals(" xxx\\u2122xxx", comment.getContent());
     }
+
+    @Test
+    public void testReplaceDuplicateJavaDocComment() {
+        // Arrange
+        CompilationUnit cu = parse("public class MyClass {" + EOL +
+                EOL +
+                "  /**" + EOL +
+                "   * Comment A" + EOL +
+                "   */" + EOL +
+                "  public void oneMethod() {" + EOL +
+                "  }" + EOL +
+                EOL +
+                "  /**" + EOL +
+                "   * Comment A" + EOL +
+                "   */" + EOL +
+                "  public void anotherMethod() {" + EOL +
+                "  }" + EOL +
+                "}" +
+                EOL);
+
+        MethodDeclaration methodDeclaration = cu.findFirst(MethodDeclaration.class).get();
+
+        // Act
+        Javadoc javadoc = new Javadoc(JavadocDescription.parseText("Change Javadoc"));
+        methodDeclaration.setJavadocComment("", javadoc);
+
+        // Assert
+        assertEquals("public class MyClass {" + EOL +
```

Comment:
Take a look at assertEqualsNoEol (or so) - building on unix and windows is painful, and has everything to do with comparing eol's :-)

Reply:
To be honest, I don't not see much of a difference between using EOL and \n everywhere :)
But I guess assertEqualsNoEol gives the freedom to copy pase in Intellij IDEA, is that why you use it?
I will change it if you like.

Reply:
Do change it - EOL is the *platform specific* end of line character, and `assertEqualsNoEol` replaces each and every eol character to \n so you can safely compare to strings with \n in it. These tests fail on Windows.

Reply:
I changed it now. Maybe I misunderstand something but I can safely say that the test was running on Windows before, as I programmed it on Windows.
Also `Utils.EOL` already uses `System.getProperty("line.separator")` which works on all platforms and `assertEqualsNoEol`, as I understand it, basically replaces the "\n" back to `Utils.EOL`, which is why I didn't really see a difference between before and after, maybe I am missing something.
Anyway, I changed it and I didn't really want to start a big discussion :)